config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
-  resources :stocks, only: [:index, :show]
+  resources :stocks, only: [:index, :show, :create]
   resources :warehouses
   resources :products
+  resources :orders, only: [:index, :show, :create]
+  patch "dispatch_order/:id", to: "orders#update"
```

Comment:
Had the thought of moving create to a CQRS style endpoint, but left it as is as create is good naming, patch/update didn't seem to match the name for what the endpoint was doing.